src/main/features/claim/views/eligibility/not-eligible.njk

```diff
@@ -20,6 +21,15 @@
           + '<a target="_blank" href="%s">form N1</a> ' + newWindow() + '.',
           { postProcess: 'sprintf', sprintf: [ n1FormUrl ]}) | safe }}
         </p>
+      {% elseif reason === NotEligibleReason.MULTIPLE_CLAIMANTS %}
+        <p>{{ t('You can’t use this service if more than one person or business is making the claim') }}</p>
+        <p>{{ t('Use <a target="_blank" href="%s">form N1 </a>' + newWindow() + ' to make your claim..',
+          { postProcess: 'sprintf', sprintf: [ n1FormUrl ]}) | safe }}</p>
```

Comment:
Do not use `sprintf` processor anymore, use interpolation provided by i18next See: 

https://www.i18next.com/interpolation.html

Reply:
As all the other reasons use sprintf, would you like me to rework them too?

Reply:
separate PR

Reply:
👍